### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Thoth1111**
### Mobile Menu

In this branch, I used Javascript and DOM manipulation to make the hamburger menu of the mobile version interactive. I did this by:

- Modifying the HTML and CSS of the navigation menu and adding transitions to activate them on a click event.
- Adding and linking a myscript.js file to contain the event listeners.
- Creating event listeners to activate and deactivate the menu on any click event.
- Modifying the HTML and CSS to closely mimic figma template three design. 
- Fixing linter errors to ensure the code is clean and in line with HTML, CSS, and JS best practices.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Thoth1111 and @HanaSabih, kindly update your app's key features and futures features in your readme

**Review by prolajumokeoni — COMMENTED** [review]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the javascript linters setup, and does not have a descriptive readme, please update the key features and future features and other sections yet to be updated in your readme

For this project both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters have to be setup
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Thoth1111** [comment]
- I've updated the ReadMe.md file in the master branch. 

- I had set up the Javascript linters prior but assumed that the linters.yml file was similar to what I had before. I have replaced it with the Javascript one since I can't have two files with the same name. EsLint is now included in the last check.

**Comment by prolajumokeoni** [comment]
Hi @Thoth1111 please update the readme in this current branch you are requesting review for, please use the [Html, CSS and javascript linters ](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

**Comment by Thoth1111** [comment]
I've changed the readme on the current branch and set up all linters now.

File: myscript.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navbar.classList.toggle('active');
+});
+
+closeBtn.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navbar.classList.toggle('active');
+});
```
Comment (Abenezer-Tilahun) [inline myscript.js:13]: - [x] When I clicked on the hamburger and close button the mobile menu appears as show below on the video  .But as stated in the project requirement When the user clicks (or taps) the hamburger button, the mobile menu appears and When the user clicks (or taps) the close (X) button, the mobile menu disappear.So Kindly fix it 

Please check this [video](https://drive.google.com/file/d/1PNp4sFDHTbio3mkH86V_x2Y43AFMYNb3/view?usp=share_link) and take the appropriate adjustment .

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Thoth1111 @HanaSabih   , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct GitHub Flow.
✔️ Professional README
✔️ Good commit messages

almost

## Required Changes ♻️

_Check the comments below the review._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: myscript.js (hunk 1 above)
Comment (Thoth1111) [reply myscript.js:13]: Thank you for the feedback. I have made the change to this now.

**Review by mckent05 — APPROVED** [review]
Hi @Thoth1111 , 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Your mobile menu works as expected 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**